### DISCUSSION TITLE
Taskbar issue

### DISCUSSION THREAD

**Discussion by AlAqsaRetoW** [builtbybel/ThisIsWin11#59] [General]
my taskbar becomes smaller after using the software. I'm using a laptop. any fix for this issue? or is there any way to reset it?

**Comment by AlAqsaRetoW** [comment]
> my taskbar becomes smaller after using the software. I'm using a laptop. any fix for this issue? or is there any way to reset it?

oh, I found the fix https://www.tomshardware.com/how-to/change-taskbar-icon-size-windows-11, just need to change the value to 1.

**Reply by anshulmaske** [reply]
Thanks for the fix 👍🏼

**Comment by Llareggub-cloud** [comment]
I want to move the taskbar to the left (vertical) side of the screen. This was available in Windows10.
